bot/cogs/antispam.py

```diff
@@ -243,9 +242,6 @@ def mod_log(self) -> ModLog:
 
     async def _process_deletion_context(self, context_id: int) -> None:
         """Processes the Deletion Context queue."""
-        log.trace("Sleeping before processing message deletion queue.")
-        await asyncio.sleep(10)
-
```

Comment:
I'm not sure this part should be deleted, as discussed in [#dev-contrib](https://discordapp.com/channels/267624335836053506/635950537262759947/644820856958222338) I believe we should wait for response from @SebastiaanZ to confirm what it is doing first.

Reply:
Comment from @SebastiaanZ in #dev-contrib:
> It should not wait 10 seconds to delete the messages after triggering, but it should take action as soon as possible
> However, since you are going to relay the attachments, that's something that could happen earlier
> The DeletionContext tracks the messages that should be deleted
> The `add` method of that class seems like a good place to relay the attachments and save the links to those
> That happens before the messages are deleted

I’m going to move it ASAP.